openfe/setup/campaigners/building_blocks.py

```diff
@@ -0,0 +1,36 @@
+from typing import Iterable, FrozenSet, Tuple
+
+from rdkit import Chem
+
+from gufe import Protocol
+
+from .. import Transformation, SmallMoleculeComponent, ProteinComponent, LigandAtomMapping, ChemicalSystem
+
+
+
+def load_files(ligand_sdf_paths:Iterable[str], receptor_pdb_path:str)-> Tuple[Iterable[SmallMoleculeComponent], ProteinComponent]:
+    small_components = []
+    for ligand_sdf in ligand_sdf_paths:
+        suppl = Chem.SDSupplier(ligand_sdf)
+        small_components.extend([SmallMoleculeComponent(rdkit=mol) for mol in suppl])
```

Comment:
I think a helper to load `SmallMoleculeComponent`s from an SDF (possibly containing many) is probable a useful function on its own, rather than combined with the 1-liner to load the protein. That said, there are questions here about how to ensure that the user intends to provide different molecules, and not isn't giving us a file with multiple conformers of a single molecule.

Reply:
I placed this code to the cli tool, I think there its a list comp.